### DISCUSSION TITLE
Embedded objects in SQL databases

### DISCUSSION THREAD

**Discussion by mwanago** [prisma/prisma#21876] [Q&A]
## Problem
Hi. I would like to create a model that contains another nested model.
```typescript
model User {
  id       Int    @id @default(autoincrement())
  name     String
  address  Address
}

model Address {
  city    String
  street  String
}
```
I don't want to use relations and create a separate table for the `Address` model. Is that currently possible with Prisma and SQL databases? This would work similarly to the embedded entities in TypeORM.
https://orkhan.gitbook.io/typeorm/docs/embedded-entities

**Comment by mwanago** [comment]
@janpio Hi. I created it as a feature request, because I could not have found this feature in the documentation.

**Comment by ludralph** [comment]
Hi @mwanago 👋 

Thank you for raising this question.

In order for our engineering team to consider this as a feature request, kindly use this [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) template and document your use case.

**Reply by janpio** [reply]
This was converted from an issue (as visible in the right sidebar).

We turned it into a discussion, so the question that Marcin poses can be answered.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
